src/components/atoms/Alert/Alert.tsx

```diff
@@ -24,29 +25,33 @@ export interface AlertProps extends React.HTMLAttributes<HTMLDivElement> {
 const Wrapper = styled.div<{ severity: Severity; inline: boolean; hasRoundedCorners: boolean }>`
   display: ${({ inline }) => (inline ? 'inline-flex' : 'flex')};
   position: relative;
-  padding: 8px 12px 8px 12px;
+  align-items: ${({ severity, theme }) => theme.alert[severity].alignItems};
+  padding: ${({ severity, theme }) => theme.alert[severity].padding ?? '8px 12px 8px 12px'};
   background: ${({ severity, theme }) => theme.alert[severity].background};
   color: ${({ severity, theme }) => theme.alert[severity].text};
-  font-size: ${typography.sizes.text.body};
-  line-height: ${typography.sizes.lineHeight.body};
-  font-family: ${typography.primary};
-  font-weight: 400;
-  border-radius: ${({ hasRoundedCorners }) => (hasRoundedCorners ? 4 : 0)}px;
+  font-size: ${({ theme }) => theme.typography.text.sizes.body ?? typography.sizes.text.body};
+  line-height: ${({ theme }) => (theme.typography.lineHeight.small ? typography.sizes.lineHeight.body : '24px')};
+  font-family: ${({ theme }) => theme.typography.primary ?? typography.primary};
+  font-weight: ${({ theme }) => theme.typography.weights.ultraBold ?? '400'};
+  border-radius: ${({ severity, theme, hasRoundedCorners }) =>
+    theme.alert[severity].borderRadius ? theme.alert[severity].borderRadius : hasRoundedCorners ? '4px' : '0px'};
   border: ${({ severity, theme }) => theme.alert[severity].border};
+  border-left: ${({ severity, theme }) => theme.alert[severity].borderLeft};
+  width: ${({ severity, theme }) => theme.alert[severity].width};
```

Comment:
why do we need fixed width/height? we should avoid setting those

Reply:
the width in JL for Alert is fixed. and the height is using hug. while Zopa's are different

Reply:
the width is not fixed for Alert v2, if you take a look at miro the width is determined by page content wrapper (you can spot that the text before submit button is same width). so we don't want to have fixed width

Reply:
didn't get whats going on with height? height is really a thing you never want to be fixed

Reply:
I am not getting much inforation from miro board. 
I can only see on the Figma that

width: Fill (612px)px;
height: Hug (64px)px;

Can you elaborate it?

Reply:
ok. I have changed to this:   
width: 'fill-available',
height: 'auto',

I guess what you are saying is it doesn't have to be the exact same size as it indicated in Figma as it adjusts based on the page itself